test/fuzz/param.ml

```diff
@@ -1,5 +1,7 @@
 let debug = true
 
-let reference_fuzzing = true
+let reference_fuzzing = false
```

Comment:
Why did you disable this ?

Reply:
Only for my local tests, to speed up execution ;)
But the timeout process seems to work not so good with the official interpreter :(